src/pages/candidate-details/candidate-details.html

```diff
@@ -34,15 +34,9 @@ <h4>Candidate details</h4>
         <ion-row class="mes-data-row-with-separator">
           <ion-col col-4>
             <label>Slot type</label>
-            <img 
```

Comment:
are we dropping the exclamation Indicator?

Reply:
Yes, we are not going to have any indicator next to the slot type. The exclamation mark will be visible next to different other fields, special requirements, entitlement check etc.